.github/workflows/stale.yml

```diff
@@ -0,0 +1,43 @@
+name: close stale issues and pr
+on:
+  schedule:
+    - cron: "30 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          debug-only: true
+          days-before-issue-stale: 28
+          days-before-issue-close: 7
+          any-of-labels:
+            [
+              "Waiting on author",
+              "Cannot reproduce 🤷",
+              "python past end of life",
+              "Won't fix/not planned",
+              "Work in progress",
```

Comment:
Maybe "Need decisions" could be added here ?

Reply:
I'd prefer to start small instead 😄